scripts/apps-version-check.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euo pipefail
+
+latest_tag=$(git describe --tags "$(git rev-list --tags --max-count=1)")
+
+bad_app_count=0
+
+while read -r app; do
+    if [ "$app" != "emqx" ]; then
+        app_path="$app"
+    else
+        app_path="."
+    fi
+    src_file="$app_path/src/$(basename "$app").app.src"
+    app_version=$(grep -E 'vsn' "$src_file" | grep -oE '[0-9]+\.[0-9]+(-alpha|-beta|-rc)?\.[0-9]+')
+    if [ -n  "$(git diff --name-only --diff-filter=ACMRTUXB "$latest_tag"...HEAD \
+                    -- $app_path/etc \
+                    -- $app_path/include \
+                    -- $app_path/priv \
+                    -- $app_path/src \
+                    -- $app_path/test )" ]; \
+    then
+        if [ "$app_version" == "${latest_tag#[e|v]}" ]; then     
```

Comment:
app versions do not follow release tag.
all we need is to check if there is an update in vsn, but which exact vsn to use should depend on review.